.test-infra/dataproc/create_flink_cluster.sh

```diff
@@ -57,6 +59,10 @@ DOCKER_INIT="$GCS_BUCKET/$INIT_ACTIONS_FOLDER_NAME/docker.sh"
 
 # Flink properties
 FLINK_LOCAL_PORT=8081
+
+# By default each taskmanager has one slot - use that value to avoid sharing SDK Harness by multiple tasks.
+FLINK_TASKMANAGER_SLOTS="${FLINK_TASKMANAGER_SLOTS:=1}"
```

Comment:
@angoenka @mxm could you take a look at the error here: https://builds.apache.org/job/beam_LoadTests_Python_GBK_Flink_Batch_PR/14/console

According to my understanding, we have now 5 workers, each one having 1 slot. The error states that there is to few resources to run. Did I miss something?

Reply:
More importantly, how can I determine how many workers with one slots should I have to succeed? Or maybe (for some reason) we cannot have 1 slot per worker? 🤔

Reply:
What parallelism do you use? Generally, the total number of task slots needs to match the parallelism.

Reply:
parralelism = 5, there are 5 Dataproc workers but I just found out that the flink.sh init action creates 4 task_managers. I think this is the culprit. I will take a look later.

Reply:
suspected line I'm talking about: https://github.com/apache/beam/blob/5ee4cf4e4880782492ec26f2b454a6df9b25f1e2/.test-infra/dataproc/init-actions/flink.sh#L112